graphql/schema.graphql

```diff
@@ -253,6 +257,10 @@ type Test {
 
   status: TestStatus!
 
+  runningTime: NonNegativeSeconds! @rename(attribute: "time")
```

Comment:
The page with the filtering doesn't like the type: 

![image](https://github.com/user-attachments/assets/fe07420e-fd8f-48a2-a9a8-3629efff1258)

Reply:
Details is an arbitrary string as far as I am aware.  Some clients who use CDash without CTest insert other information in the details field.

Reply:
@josephsnyder Fixed.  Adding tests for the both the filter interface and the test page (and setting up the necessary infrastructure to do so) is on my list of things to to once 3.5 is released.  Hopefully that will catch errors like this.